### PR TITLE
fixed Issue13 and a complementary fix

### PR DESCRIPTION
**Pull request by CakeCrusher**
I also made it so that there is only one possible empty step when iframe loaded this should be consistent with issue#20 when it is created

### CODE REVIEW

**Comment by baptisteArno** [comment]
Can you link the corresponding issue?

File: src/service/helperFunctions.tsx [hunk 1]
```diff
@@ -146,3 +146,41 @@ export const findUniqueSelector = (
...
+export const isStepInActionProcess = (step: Step): boolean => {
+  if (
+    (step.action === StepAction.RECORD_CLICKS_KEYS && !step.recordedClicksAndKeys!.length) ||
+    (step.action !== StepAction.RECORD_CLICKS_KEYS && !step.tagName)
+  ) {
+    return true
+  } else {
+    return false
+  }
+}
```
Comment (baptisteArno) [inline src/service/helperFunctions.tsx:159]: You can make it more elegant:
```ts
export const isStepInActionProcess = (step: Step): boolean => (step.action === StepAction.RECORD_CLICKS_KEYS && !step.recordedClicksAndKeys!.length) ||
    (step.action !== StepAction.RECORD_CLICKS_KEYS && !step.tagName)
```

File: src/service/helperFunctions.tsx [hunk 2]
```diff
@@ -146,3 +146,41 @@ export const findUniqueSelector = (
...
+
+// for future use
+export const stepAlreadyInProcess = (steps: Step[]): boolean => {
+  const inProcessSteps = steps.filter(step => isStepInActionProcess(step))
+  if (inProcessSteps.length) {
+    return true
+  } else {
+    return false
+  }
+}
```
Comment (baptisteArno) [inline src/service/helperFunctions.tsx:186]: You can remove `reduceInProcessSteps` & `stepAlreadyInProcess`. Last PR is forbidding having multiple "empty" steps

**Review by baptisteArno — DISMISSED** [review]

**Comment by CakeCrusher** [comment]
Fixed it up according to your requested changes

**Pull request merged**